### DISCUSSION TITLE
username and password for web ui after docker installation

### DISCUSSION THREAD

**Discussion by langlichong** [librenms/docker#425] [Q&A]
when install successfully by docker , the first time to access web ui then be ask for a username and password , what are they?
after search online , try the librenms , no work , and the official docs has no nothing about the web ui login credentias

**Comment by crazy-max** [comment]
There is none, you need to open the webui and set up the admin account yourself, see https://github.com/librenms/docker/pull/280

**Comment by murrant** [comment]
If you are using a database that has already been set up, you will already have a user.  If you don't know what that is, use `lnms user:add --role=admin` to create a new admin user.
